packages/flutter/test/widgets/selectable_region_test.dart

```diff
@@ -485,6 +485,54 @@ void main() {
     });
   });
 
+  testWidgets('Can extend MultiStaticSelectableSelectionContainerDelegate', (WidgetTester tester) async {
+    SelectedContent? content;
+    final FocusNode focusNode = FocusNode();
+
+    // Inserts a new line between selected content of children selectables.
+    final ColumnSelectionContainerDelegate selectionDelegate = ColumnSelectionContainerDelegate();
+
+    addTearDown(focusNode.dispose);
+    addTearDown(selectionDelegate.dispose);
+
+    await tester.pumpWidget(
+      MaterialApp(
+        home: SelectableRegion(
+          onSelectionChanged: (SelectedContent? selectedContent) => content = selectedContent,
+          focusNode: focusNode,
+          selectionControls: materialTextSelectionControls,
+          child: Center(
+            child: SelectionContainer(
+              delegate: selectionDelegate,
```

Comment:
Could this be done near the top of the widget tree for a big app with a single SelectionContainer, so that nearly everything has this newline behavior? Or would you need to do a bunch of SelectionContainers?

Reply:
It would still need multiple SelectionContainers because `getSelectedContent` does not propagate down the tree.